apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -444,6 +450,23 @@ type AutoscalerSpec struct {
 	TargetMemoryUtilization *int32 `json:"targetMemoryUtilization,omitempty"`
 }
 
+// PodDisruptionBudgetSpec defines the OpenTelemetryCollector's pod disruption budget specification.
+type PodDisruptionBudgetSpec struct {
+	// An eviction is allowed if at least "minAvailable" pods selected by
+	// "selector" will still be available after the eviction, i.e. even in the
+	// absence of the evicted pod.  So for example you can prevent all voluntary
+	// evictions by specifying "100%".
+	// +optional
+	MinAvailable *intstr.IntOrString `json:"minAvailable,omitempty"`
```

Comment:
does one of these need to be set in order to be accepted?

Reply:
What do you mean?

Reply:
does someone who wants to use a PDB need to set either min or max available for the kube api to accept the object?

Reply:
No no, the must set only one of them, they are mutually exclusive. If they set both, the admission webhook will reject the CR: https://github.com/open-telemetry/opentelemetry-operator/pull/2141/files#diff-f7d2d1348f61f254e8b086ada3e9ad6ccc705b9cbbd171065911fd75b38bf495R251-R255.

Reply:
ah i missed it in my initial review, thank you! This LGTM.

Reply:
no worries, it wasn't in your initial review, I added it due to your feedback 😄